src/gateway/WebSocketClient.ts

```diff
@@ -0,0 +1,128 @@
+import WebSocket from 'ws';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { EVENTS, Payload } from './constants';
+import { GATEWAY_OPCODES, GATEWAY_CLOSE_EVENT_CODES } from '../util/Constants';
+const IDENTIFY_TIMEOUT = 4000;
+
+export default class WebSocketClient {
+  private _intents: number;
+  private _token: string;
+  private _url: string;
+  private heartBeatInterval!: NodeJS.Timeout;
```

Comment:
Any reasons those properties are not prefixed?

Reply:
changed